proxy/src/http/control.rs

```diff
@@ -131,13 +132,12 @@ mod handler {
     }
 
     /// Reset the coco state by creating a new temporary directory for the librad paths.
-    pub async fn nuke_coco(librad_paths: Arc<RwLock<Paths>>) -> Result<impl Reply, Rejection> {
-        let dir = tempfile::tempdir().expect("tmp dir creation failed");
-        let new = Paths::from_root(dir.path()).expect("unable to get paths");
-
-        let mut paths = librad_paths.write().await;
+    pub async fn nuke_coco(_peer: http::Shared<coco::UserPeer>) -> Result<impl Reply, Rejection> {
```

Comment:
What we want here is to reset that the coco state, meaning having a new librad monorepo tree which is fresh and backed by a temp directory. We used to swap the `Paths`, dunno how to do it now, replace the `UserPeer` entirely maybe?